batch_job/src/work_queue_factory.c

```diff
@@ -199,6 +208,10 @@ int manager_workers_needed_by_resource(struct jx *j) {
 		needed = MAX(needed, DIV_INT_ROUND_UP(tasks_total_disk, disk));
 	}
 
+	if(gpus > 0 && tasks_total_gpus > 0) {
```

Comment:
gpus should also be added to configuration file (around line 704)

Reply:
Indeed, thanks!